frontend/src/components/LendingForm.js

```diff
@@ -57,30 +58,36 @@ const LendingForm = ({walletId}) => {
             const backendUrl = process.env.BACKEND_URL || 'http://127.0.0.1:8000';
             console.log("BACKENDURL", backendUrl)// Replace with your backend URL
             console.log("Query Params:", queryParams);
-            const response = await fetch(`${backendUrl}/transaction-data?${queryParams}`, {
-                method: 'GET',
+            // const response = await fetch(`${backendUrl}/transaction-data?${queryParams}`, {
+            //     method: 'GET',
+            //     headers: {
+            //         'Content-Type': 'application/json',
+            //     },
+            // });
+
+            const response = await axios.get(`${backendUrl}/transaction-data`, {
```

Comment:
axios should not be accessed directly, instead please abstract base instance of axios into a separate file (https://axios-http.com/docs/instance), configure baseUrl, export and use this base instance across the app and remove all direct references of backendUrl in that requests (baseUrl should be used)